fix build format

acmol opened the pull request.

acmol committed:
fix build format

acmol committed:
add note for docker user

advancedxy commented:
![image](https://user-images.githubusercontent.com/807537/33312450-e42fe3e2-d462-11e7-905e-25aa1eb00c89.png)

Is this line of code rendered correctly?

acmol committed:
fix doc format

I don't know whether it is the wrong with the last, but this is the correct format.
